integration-cli/requirements.go

```diff
@@ -40,7 +40,7 @@ var (
 		func() bool {
 			// Set a timeout on the GET at 15s
 			var timeout = time.Duration(15 * time.Second)
-			var url = "https://hub.docker.com"
+			var url = "http://www.baidu.com"
```

Comment:
not sure this should be changed? (also this changes from a https:// to a http:// domain)